### PR TITLE
docs: polishing authz-keycloak's docs

### PR DESCRIPTION
**Pull request by Hwting**
### Description

docs: polishing authz-keycloak plugin's docs
Fixes # (issue)

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [x] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

File: docs/zh/latest/plugins/authz-keycloak.md [hunk 1]
```diff
@@ -81,6 +81,19 @@ curl http://127.0.0.1:9080/apisix/admin/routes/5 -H 'X-API-KEY: edd1c9f034335f13
...
 
+获取`{JWT Token}`
+
+```shell
+curl \
+  -d "client_id=<YOUR_CLIENT_ID>" \
+  -d "username=<YOUR_USERNAMED>" \
+  -d "password=<YOUR_PASSWORD>" \
+  -d "grant_type=password" \
+  "http://localhost:8080/realms/${realm}/protocol/openid-connect/token"
```
Comment (starsz) [inline docs/zh/latest/plugins/authz-keycloak.md:92]: Port is 8080?

Comment (Hwting) [reply docs/zh/latest/plugins/authz-keycloak.md:92]: KEYCLOAK_HTTP_PORT: Keycloak HTTP port. Default: 8080，Or change to <YOUR_KEYCLOAK_HOST>

File: docs/zh/latest/plugins/authz-keycloak.md [hunk 2]
```diff
@@ -57,15 +57,15 @@ title: authz-keycloak
 
 ## 如何启用
 
-创建一个 `route` 对象，并在该 `route` 对象上启用 `authz-keycloak` 插件：
+创建一个 `route` 对象，并在该 `route` 对象上启用 `authz-keycloak` 插件,`${realm}`是`Keycloak`中的`realm`名称：
```
Comment (zaunist) [inline docs/zh/latest/plugins/authz-keycloak.md:60]:
```suggestion
创建一个 `route` 对象，并在该 `route` 对象上启用 `authz-keycloak` 插件, `${realm}` 是 `Keycloak` 中的 `realm` 名称：
```

File: docs/zh/latest/plugins/authz-keycloak.md [hunk 3]
```diff
@@ -81,6 +81,19 @@ curl http://127.0.0.1:9080/apisix/admin/routes/5 -H 'X-API-KEY: edd1c9f034335f13
 
 ## 测试插件
 
+获取`{JWT Token}`
```
Comment (zaunist) [inline docs/zh/latest/plugins/authz-keycloak.md:84]:
```suggestion
获取 `{JWT Token}`
```

Comment (zaunist) [reply docs/zh/latest/plugins/authz-keycloak.md:84]: Should be a black space between "``" block and word.

**Review by starsz — DISMISSED** [review]

**Review by zaunist — DISMISSED** [review]

**Review by juzhiyuan — COMMENTED** [review]
Hi, could you also update the English version? Thanks!

**Review by starsz — APPROVED** [review]

**Pull request merged**